### PR TITLE
Preparedness page infrastructure

### PR DESCRIPTION
**Pull request by nein09**
This is a start to the Preparedness page (https://trello.com/c/twgEaKps) - it loads the data from a spreadsheet into Django, and creates the page url, route and views.

In the process of doing this, I extracted a lot of common code from the snugget loader as well as the front-end template, and put them in places where they could be shared.

I also included the work for this in here (https://trello.com/c/J6IusJCX)

Next step: styling because it looks like this right now:
image

### CODE REVIEW

File: load_helpers.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        tag_opening_no_attr = "<" + tag + ">"
+        tag_opening_with_attr = "<" + tag + " "
+        tag_closing = "</" + tag + ">"
+        tag_openings = row[key].count(tag_opening_no_attr) + row[key].count(tag_opening_with_attr)
+        tag_closings = row[key].count(tag_closing)
+        if tag_openings > tag_closings:
+          print("WARNING: In row", str(rowCount), key, "has", str(tag_openings), "opening <" + tag + "> tag[s] but only", str(tag_closings), "closing tag[s].")
+
+
+def allRequiredFieldsPresent(requiredFields, optionalFields, row, rowCount):
```
Comment (eldang) [inline load_helpers.py]: I don't think we ever actually use `requiredFields`, and instead this function assumes that any field not explicitly marked optional is required.  Unless I've missed something, let's remove that argument and the variable declarations altogether.

File: load_helpers.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  def item(i, j):
+    return (
+      cleanValue(sheet.cell(row=1, column=j).value),
+      cleanValue(sheet.cell(row=i, column=j).value)
+    )
+
+  return (dict(item(i,j) for j in range(1, cols)) for i in range(2, rows))
+
+
+def runLoader(config):
```
Comment (eldang) [inline load_helpers.py:72]: Oh, I like this way of structuring it.

File: snugget_load.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 requiredFields = ['shapefile', 'section', 'subsection']
 # all other fields in snuggetFile are required. The empty string is to deal with Excel's charming habit of putting a blank column after all data in a CSV.
```
Comment (eldang) [inline snugget_load.py]: The second sentence of this comment no longer applies - let's just cut it so it doesn't confuse us later.

**Comment by eldang** [comment]
Hooray!  I like how this seems to simplify the code overall even as it adds some functionality.

If I'm right about `requiredFields` not being used, then I think it is worth cleaning up the declarations and references to that.  Other than that I think this is ready to merge.

**Comment by nein09** [comment]
I'll do some cleanup (and then open a matching PR on PDX-Ready).

File: load_helpers.py (hunk 2 above)
Comment (nein09) [reply load_helpers.py:72]: 🎉

**Pull request merged**